### PR TITLE
Dec stable Release CHANGELOG.md

### PR DESCRIPTION
**Pull request by summzhan**

### CODE REVIEW

File: packages/vscode-extension/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,54 @@
 # Changelog
 
+## 5.4.0 - Dec 12, 2023
+
+This minor version update of Teams Toolkit includes new features and bug fixes based on customers feedback. The new features include the integrated Adaptive Cards Previewer, the Teams app test tool for Teams Bot app and new project templates for AI Assistant Bot, and refreshed Sample Gallery look. We previously shared these incremental changes in the prerelease version and through a series of blog posts:
```
Comment (therealjohn) [inline packages/vscode-extension/CHANGELOG.md:5]: "This minor version update of Teams Toolkit includes new features and bug fixes based on your feedback. The new features include the Adaptive Card Previewer, Teams App Test Tool, new project templates for AI Assistant Bot, and a refreshed Sample Gallery. We previously shared these incremental changes in the prerelease version and through a series of blog posts:"

File: packages/vscode-extension/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,54 @@
...
+
+- [October Prerelease](https://devblogs.microsoft.com/microsoft365dev/teams-toolkit-update-october-2023): Search sample app, new Contoso Retail Dashboard sample app, Teams Toolkit CLI improvement.
+- [November Prerelease](https://devblogs.microsoft.com/microsoft365dev/teams-toolkit-for-visual-studio-code-update-november-2023): Brand-new AI Assistant Bot App template, Teams app test tool, integrated Adaptive Card Previewer, Refreshed look for Sample Gallery etc.
+
+We've listened to your feedback and included these additional new features, enhancements, and bug fixes to this release.
+
+New features:
+
+- AI Assistant Bot App template is powered by the Teams AI library and the OpenAI Assistants API, this template demonstrates how to build an intelligent chatbot within Teams.
```
Comment (therealjohn) [inline packages/vscode-extension/CHANGELOG.md:14]: AI Assistant Bot App template using Teams AI library and the OpenAI Assistants API. This template demonstrates how to build an intelligent chatbot within Teams.

File: packages/vscode-extension/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,54 @@
...
+  ![AI-assistant-bot](https://github.com/OfficeDev/TeamsFx/assets/113089977/13877e7b-cffb-4717-9d1e-587478a43e5d)
+- Teams App Test Tool empowers developers to debug, test, and iterate on the Teams Bot app design within a web-based chat environment.
```
Comment (therealjohn) [inline packages/vscode-extension/CHANGELOG.md:16]: Teams App Test Tool removes the need for a lot of the setup to run and debug a Teams bot in a web-based chat environment that is similar to Teams.

File: packages/vscode-extension/CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,54 @@
...
+  ![Teams app test tool](https://github.com/OfficeDev/TeamsFx/assets/113089977/7f6bcf56-27b3-4a7b-a160-64ab847efd92)
+- Preview and edit Adaptive Cards effortlessly with the integrated Adaptive Card Previewer directly within Teams Toolkit.
```
Comment (therealjohn) [inline packages/vscode-extension/CHANGELOG.md:18]: Preview and edit Adaptive Cards effortlessly using the Adaptive Card Previewer now included in Teams Toolkit.

File: packages/vscode-extension/CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,54 @@
...
+  ![Adaptive Card Previwer](https://github.com/OfficeDev/TeamsFx/assets/113089977/8062f237-d640-45b2-a7b2-81a0a4b0aa10)
+- Sample Gallery new design offers grid view and list view, making it easy to navigate among samples. You can also search samples or filter samples by type, capability, and programming language.
```
Comment (therealjohn) [inline packages/vscode-extension/CHANGELOG.md:20]: Navigate quickly through samples using list or grid views. You can also search samples or filter samples by type, capability, and programming language.

File: packages/vscode-extension/CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,54 @@
...
+  ![New sample gallery UI](https://github.com/OfficeDev/TeamsFx/assets/113089977/cbb18477-1f6d-4ab5-a947-1840d3290e40)
+- Developing with Microsoft Copilot? We’ve added a helpful license check UI to ensure that your account has the necessary Microsoft Copilot licenses.
+  ![Copilot license check](https://github.com/OfficeDev/TeamsFx/assets/113089977/78fa1fc5-0e19-40f0-b324-65583485f743)
+
+Enhancement:
+
+- New samples arrivals at Sample Gallery:
```
Comment (therealjohn) [inline packages/vscode-extension/CHANGELOG.md:27]: New samples in the Sample Gallery:

**Review by therealjohn — CHANGES_REQUESTED** [review]

File: packages/vscode-extension/CHANGELOG.md (hunk 1 above)
Comment (summzhan) [reply packages/vscode-extension/CHANGELOG.md:5]: updated

File: packages/vscode-extension/CHANGELOG.md (hunk 2 above)
Comment (summzhan) [reply packages/vscode-extension/CHANGELOG.md:14]: updated

File: packages/vscode-extension/CHANGELOG.md (hunk 3 above)
Comment (summzhan) [reply packages/vscode-extension/CHANGELOG.md:16]: updated

File: packages/vscode-extension/CHANGELOG.md (hunk 4 above)
Comment (summzhan) [reply packages/vscode-extension/CHANGELOG.md:18]: updated

**Review by sffamily — APPROVED** [review]

**Pull request merged**